docs/services/github.md

```diff
@@ -75,8 +75,11 @@ template.app-deployed: |
       environmentURL: "https://{{.app.metadata.name}}.example.com"
       logURL: "{{.context.argocdUrl}}/applications/{{.app.metadata.name}}?operation=true"
       requiredContexts: []
+      autoMerge: true
 ```
 
 **Notes**:
 - If the message is set to 140 characters or more, it will be truncated.
 - If `github.repoURLPath` and `github.revisionPath` are same as above, they can be omitted.
+- By default, github deployments utilize automerge to ensure the requested ref is up to date with the default branch.
+  Setting this option to `false` is required if you would like to deploy older refs in your default branch.
```

Comment:
Can we also mention that this parameter is optional and by default it is false, i would write opposite , like: Setting this option to `true` is required ....

Reply:
Sorry for the delay I realized that you're right, this PR set the value to false by default however the upstream API is true by default.  I had to modify the PR and language in the doc to reverse the behavior. I hope the language in the docs is more clear now. Thanks!